PR fails on Travis-CI:
```

grouping_test.cpp:363: error: check failure #2: 'testAggregation(ctx, request, expect)' in thread 'master' (grouping_test)
    STATE[0]: 'testAggregationSimple()' (grouping_test.cpp:1964)
grouping_test: info:  imported 291 passed check(s) from 1 thread(s)
grouping_test: info:  summary --- 291 check(s) passed --- 2 check(s) failed
grouping_test: info:  CONCLUSION: FAIL
```

--- a/searchlib/src/tests/grouping/grouping_test.cpp
+++ b/searchlib/src/tests/grouping/grouping_test.cpp
@@ -313,10 +313,9 @@ Test::testAggregationSimple()
     ctx.add(FloatAttrBuilder("float").add(3).add(7).add(15).sp());
     ctx.add(StringAttrBuilder("string").add("3").add("7").add("15").sp());
 
-    char strsum[3] = {-101, '5', 0};
-    testAggregationSimpleSum(ctx, SumAggregationResult(), Int64ResultNode(25), FloatResultNode(25), StringResultNode(strsum));
-    testAggregationSimpleSum(ctx, MinAggregationResult(), Int64ResultNode(3), FloatResultNode(3), StringResultNode("15"));
-    testAggregationSimpleSum(ctx, MaxAggregationResult(), Int64ResultNode(15), FloatResultNode(15), StringResultNode("7"));
+    TEST_DO(testAggregationSimpleSum(ctx, SumAggregationResult(), Int64ResultNode(25), FloatResultNode(25), StringResultNode("25")));
+    TEST_DO(testAggregationSimpleSum(ctx, MinAggregationResult(), Int64ResultNode(3), FloatResultNode(3), StringResultNode("15")));
+    TEST_DO(testAggregationSimpleSum(ctx, MaxAggregationResult(), Int64ResultNode(15), FloatResultNode(15), StringResultNode("7")));
 }
 
 #define MU std::make_unique

